adaptive_breakpoints/lib/src/adaptive_breakpoints.dart

```diff
@@ -13,19 +13,23 @@ import 'package:flutter/material.dart';
 /// Extra large represents large computer screens.
 ///
 /// https://material.io/design/layout/responsive-layout-grid.html#breakpoints
-class AdaptiveWindowType {
-  const AdaptiveWindowType._({
-    required this.name,
-    required this.relativeSize,
-    required this.widthRangeValues,
-    required this.heightLandscapeRangeValues,
-    required this.heightPortraitRangeValues,
-  });
 
+enum AdaptiveWindowType {
   /// Name based on the [AdaptiveWindowType].
   ///
   /// Can be: xsmall, small, medium, large or xlarge
-  final String name;
```

Comment:
Please add trailing commas to make this easier to read

Reply:
Thanks. Fixed.